package.json

```diff
@@ -45,9 +45,9 @@
         "release:major": "npm run exec -- ./scripts/release.ts --release-as major",
         "*** Cypress ***": "",
         "cypress:open": "cypress open --browser chrome --project ./projects/demo-integrations",
-        "cypress:open:demo:next": "cypress open --browser chrome --project ./projects/demo-integrations --config baseUrl=\"https://taiga-ui.dev/next/\"",
+        "cypress:open:demo:next": "cypress open --browser chrome --project ./projects/demo-integrations --config baseUrl=\"http://localhost:5555\"",
```

Comment:
**nit:**
Can we make `http://localhost:5555` as environment variable to reuse in all 3 places ?